keps/sig-architecture/4330-compatibility-versions/README.md

```diff
@@ -610,20 +627,68 @@ The storage version of each group-resource is the newest
 
 ### API availability
 
-Similar to feature flags, all APIs group-versions declarations will be modified
-to track which Kubernetes version the API group-versions are introduced (or
-removed) at.
+Ideally, similar to feature flags, all API's group-version-resource declarations should be modified
+to track which Kubernetes version the GVRs are introduced (or
+removed) at: if an API is introduced after (or removed before) the emulation version, it should not be available at the emulation version. 
+
+But in practice, that would make the controller code changes intractable if an API is graduating from Beta to GA and the controller wants to use newer API. For example, to graduate Multiple Service CIDRs to GA, normally the controller code change would look like:
```

Comment:
```suggestion
But in practice, that would make the controller code changes unreasonably burdonsome if an API is graduating from Beta to GA and the controller wants to use newer API. For example, to graduate Multiple Service CIDRs to GA, normally the controller code change would look like:
```

(It's pretty clear to me that this is possible, I just don't feel comfortable asking people to carry the burden)